src/ucp/core/ucp_mm.c

```diff
@@ -594,6 +594,11 @@ ucs_status_t ucp_mem_type_reg_buffers(ucp_worker_h worker, void *remote_addr,
         goto out;
     }
 
+    if (*md_map == 0) {
+        status = UCS_ERR_NO_MEMORY;
```

Comment:
maybe we should not pass HIDE_ERRORS flag? at least in the context of dt_pack/unpack?

Reply:
yep, will work, and then ucp_mem_type_reg_buffers may be left unchanged